README.md

```diff
@@ -8,5 +8,7 @@ Permissions given to a user for a folder are inherited by all its children.
 
 You can chat with us, share your feedback and ask questions on our [Gitter chat](https://gitter.im/jenkinsci/role-strategy-plugin).
 We share the channel with the [Role Strategy Plugin](https://github.com/jenkinsci/role-strategy-plugin)
-because this plugin also follows a similar, though simplified, 'role-based' model and was created to 
+because this plugin also follows a similar, though simplified, 'role-based' model and was created to
 avoid the performance penalty of using regular expressions.
+
```

Comment:
It would be nice to have few screenshots or feature summaries on the landing page so that there is more details on thefront page